.eslintrc.js

```diff
@@ -51,64 +50,37 @@ module.exports = {
         ],
       },
     ],
-    'nonblock-statement-body-position': 'error',
-    // Airbnb restricts isNaN and isFinite which are necessary for IE 11
-    // we have to be disciplined about the usage and ensure the Number type for its
-    // arguments
-    'no-restricted-globals': ['error'].concat(confusingBrowserGlobals),
+    'no-constant-condition': 'error',
+    'no-prototype-builtins': 'off', // Use the proptype inheritance chain
     'no-underscore-dangle': 'error',
+    'nonblock-statement-body-position': 'error',
     'prefer-arrow-callback': ['error', { allowNamedFunctions: true }],
     'prefer-destructuring': 'off', // Destructuring harm grep potential.
-
-    'jsx-a11y/label-has-associated-control': 'off',
-    'jsx-a11y/label-has-for': 'off', // deprecated
-    'jsx-a11y/no-autofocus': 'off', // We are a library, people do what they want.
-
+    '@typescript-eslint/dot-notation': 'off', // Too slow
```

Comment:
Have you figured out why these are to slow (specific file? eslint issue? etc)?

Reply:
No but I can add a TODO. I know that the rule to work need accessed to the tsconfig.